content/articles/run-length-encoding/index.md

```diff
@@ -0,0 +1,150 @@
+### Introduction
+
```

Comment:
Give 2 - 3 lines quick explanation of what run length encoding is, where do we use it, and why is it used.

The very first paragraph will be displayed along with the hero image. So, make sure to attract the readers to read your article.